vapi/library/library.go

```diff
@@ -80,8 +80,11 @@ func (c *Manager) FindLibrary(
 }
 
 // CreateLibrary creates a new library with the given Type, Name,
-// Description, and CategoryID.
+// Description, and CategoryID.  Type defaults to "LOCAL".
```

Comment:
Hi @dougm,

I appreciate you catching this and setting a sane default, but I'm not sure about this long-term. The goal is to introduce `SUBSCRIBED` at a later date, and I'm not sure if it makes sense to default to a type here. @markpeek, what do you think? On the other hand, we could let the _defaults to LOCAL_ logic live in the CLI and leave this _as-is_ for now.

Reply:
I agree, I'll remove this and just set in the cli command.